data-host/library/src/popup/components/DataScreen/index.tsx

```diff
@@ -105,9 +105,7 @@ function DataScreen() {
       <Address address={address} />
       <Button
         onClick={() =>
-          activityStack.push(
-            <SendTokens onFinish={() => activityStack.pop()} />,
-          )
+            navigate(RoutesPaths.SEND)
```

Comment:
Formatting should put this one the same line.

`onClick={() => navigate(RoutesPaths.SEND)}`